devtools/common/src/main/resources/templates/dockerfile-native.ftl

```diff
@@ -5,7 +5,7 @@
 #
 # Then, build the image with:
 #
-# docker build -f src/main/docker/Dockerfile -t quarkus/${project_artifactId} .
```

Comment:
Same as the previous comment, we should add a line (first line) explaining that this runs the application in native mode.

Reply:
Done